### PR TITLE
Feature/connect to backend

### PR DESCRIPTION
**Pull request by alexthompson207**
## Is this a fix or a feature?

This is a feature.

## What is the change?

Create an `apiCalls.ts` file to hold 2 GET and 2 POST requests for connecting to our backend.

## What does it fix?

N/A

## Where should the reviewer start?

In the `apiCalls.ts` file, look at the 4 fetch requests. Then look at the new state in the `App` component and the 'MatchedRides` component

## How should this be tested?

run `npm run cypress`

### CODE REVIEW

File: src/components/Registration/Registration.tsx [hunk 1]
```diff
@@ -21,9 +22,38 @@ const Registration: FC<RegistrationProps> = (props) => {
...
+        destination,
+        departure_time: departTime,
+        days: Object.keys(days).filter(el => days[el] === true)
+      }
+      console.log(routeData)
+      postRouteData(routeData)
+        .then(response => {
+          if (response.data.attributes) {
+            <Redirect to='/matched-routes' />
+          }
```
Comment (itsnameissteven) [inline src/components/Registration/Registration.tsx]: This needs a tiny change and then all of our api calls should be working

**Review by itsnameissteven — CHANGES_REQUESTED** [review]
Great Job on this. huge help getting all the api calls working. one small change and this will be ready to merge

File: src/components/Registration/Registration.tsx (hunk 1 above)
Comment (alexthompson207) [reply src/components/Registration/Registration.tsx]: Good catch fixing this!

**Review by itsnameissteven — APPROVED** [review]
Thanks for fixing that

**Pull request merged**